src/styles/blocks/catalog.scss

```
@@ -6,6 +6,8 @@
   margin: 0 auto;
   padding: 50px 0;
 
+  transition-duration: $transition-time;
```

```suggestion
  transition color $transition-time, transform $transition-time;
```
should be in `&__card`. fix for `&__link` also